webpack.config.js

```diff
@@ -0,0 +1,72 @@
+const webpack = require("webpack");
+const path = require("path");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+
+const package = require("./package");
+const widgetName = package.widgetName;
+
+const widgetConfig = {
+    entry: `./src/${widgetName}/widget/${widgetName}.ts`,
+    output: {
+        path: path.resolve(__dirname, "dist/tmp"),
+        filename: `src/${widgetName}/widget/${widgetName}.js`,
+        libraryTarget:  "amd"
+    },
+    resolve: {
+        extensions: [ ".ts", ".js", ".json" ],
+        alias: {
+            "tests": path.resolve(__dirname, "./tests")
+        }
+    },
+    module: {
+        rules: [
+            { test: /\.ts$/, use: "ts-loader" },
+            { test: /\.scss$/, loader: ExtractTextPlugin.extract({
+                fallback: "style-loader",
+                use: "css-loader!sass-loader"
+            }) }
+        ]
+    },
+    devtool: "source-map",
+    externals: [
+        "mxui/widget/_WidgetBase",
+        /^(dojo|dijit)/i,
+    ],
```

Comment:
One liner `externals: [ /^mxui\/|^mendix\/|^dojo\/|^dijit\// ],`
Externals are case sensitive `/i` so this is not good.